docker/odoo/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ocaimages/oca-ubuntu-base
+
+MAINTAINER "Vauxoo"
+
+#  Fix log odoo warning "unable to set lang fr_FR"
+RUN locale-gen fr_FR \
+    && locale-gen en_US.UTF-8 \
+    && dpkg-reconfigure locales \
+    && update-locale LANG=en_US.UTF-8 \
+    && update-locale LC_ALL=en_US.UTF-8 \
+    && echo 'LANG="en_US.UTF-8"' > /etc/default/locale
```

Comment:
@yvaucher this is a nice solution  when testing o using a single session, but if you use docker-enter to interact with a docker container some env vars are not available for the session so is better for us to define it system wide, that's why we use this way instead of defining an ENV